src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -189,4 +201,37 @@ private function getCountryCodeByEnglishCountryName($name)
 
         return $countryCode;
     }
+
+    /**
+     * @Given there is an administrator account
+     */
+    public function thereIsAdministratorAccount()
+    {
+        /** @var UserInterface $user */
+        $user = $this->userFactory->createNew();
+        $customer = $this->customerFactory->createNew();
+        $customer->setFirstName('Admin');
+        $customer->setLastName('Admin');
+
+        $user->setCustomer($customer);
+        $user->setUsername('admin@test.com');
+        $user->setEmail('admin@test.com');
+        $user->setPlainPassword('pswd1234');
+        $user->addRole('ROLE_ADMINISTRATION_ACCESS');
```

Comment:
All the steps above should be moved to `createDefaultAdmin` method in `TestUserFactory`.